### PR TITLE
installed packages versions updated

### LINKED ISSUE #320: Update version numbers resulting from installs
The version numbers reported in the "Installing Keras and other dependencies" episode are old now.

I get the following under Python 3.11

sklearn version:  1.2.1
seaborn version:  0.12.2
pandas version:  1.5.3
Keras version:  2.12.0
Tensorflow version:  2.12.0-rc1

We should update this to reflect newer versions.

### PR DESCRIPTION
**Pull request by miguel-mx**
Fixes #320 

At the time, the latest package versions are:

sklearn version:  1.2.2
seaborn version:  0.12.2
pandas version:  1.5.3
Keras version:  2.12.0
Tensorflow version:  2.12.0

image

### CODE REVIEW

**Comment by miguel-mx** [comment]
https://github.com/carpentries-incubator/deep-learning-intro/issues/320

**Review by svenvanderburg — APPROVED** [review]
Thank you so much @miguel-mx 🙏 Wow, these versions were truly outdated...

**Comment by svenvanderburg** [comment]
Great @miguel-mx, we will try to label some more `good-first-issue` issues, but feel free to pick up anything you find interesting.
